### PR TITLE
#406 fixed nav bar

### PR DESCRIPTION
**Pull request by NavAneEth777**
# 406 
Added leaderboard heading in the nav section...

### CODE REVIEW

File: apps/host/src/components/common/navbar/navbar.tsx [hunk 1]
```diff
@@ -21,6 +29,10 @@ function Navbar({ children, title }: { children?: React.ReactNode; title?: strin
         <span>{title ?? 'Frontend Mini Challenges'}</span>
       </a>
 
+      <div className={styles.logo}>
```
Comment (sadanandpai) [inline apps/host/src/components/common/navbar/navbar.tsx:32]: can we change the class name from `logo` to something relevant

Comment (sadanandpai) [inline apps/host/src/components/common/navbar/navbar.tsx:32]: let us render this element only if it is not empty

**Review by sadanandpai — CHANGES_REQUESTED** [review]
We dont wanna show FMC in the title again in challenges page

image

**Comment by NavAneEth777** [comment]
once check... i changed it..

**Review by sadanandpai — APPROVED** [review]

**Pull request merged**